src/commands/debug_files/find.rs

```diff
@@ -149,6 +149,7 @@ fn find_ids(
                 DifType::SourceBundle => find_ids_for_sourcebundle(&dirent, &remaining),
                 DifType::Breakpad => find_ids_for_breakpad(&dirent, &remaining),
                 DifType::Proguard => find_ids_for_proguard(&dirent, &proguard_uuids),
+                DifType::JvmBased => find_ids_for_sourcebundle(&dirent, &remaining),
```

Comment:
I'm not sure about the whole `jvmbased` naming. We dont use `PdbBased`, `ElfBased`, `ProguardBasd` etc. Might use `JvmBundle` or just `Jvm` instead? wdyt @Swatinem

Reply:
Yeah I guess there's little risk in people actually thinking they need to upload a JVM. Can rename it here and in other PRs as this was suggested before. Happy to rename and make it easier to use.

Reply:
Well, naming is only one of the hardest things in software… :-D I’m okay with the naming. It clearly states that we have are dealing with any of the myriad of languages that compile to java bytecode and run on the jvm.